test/e2e/shared/defaults.go

```diff
@@ -154,6 +154,13 @@ func getLimitedResources() map[string]*ServiceQuota {
 		QuotaCode:           "L-E9E9831D",
 		DesiredMinimumValue: 20,
 	}
+
+	serviceQuotas["ec2"] = &ServiceQuota{
```

Comment:
crit. `serviceQuotas` is a map, it is being overwritten here as we already set it for regular instances above. Need to have a different name for normal instances and GPU ones.